### PR TITLE
feat: [#188079116] add license status to license calendar events

### PR DESCRIPTION
**Pull request by FarazA22**
### Approach

Note: It might be helpful to review each commit individually.

https://www.pivotaltracker.com/story/show/188259811

The nonessential question was previously created but was not mapped to the correct industry. I went ahead and added the nonessential question to the entertainment industry. Separately, I added the license name to the appropriate task and license calendar event files. I also created a test to validate the list of task ids that render the license task for completeness.

https://www.pivotaltracker.com/story/show/188339189

We noticed that there are many users that have multiple records with the same business name and address combo in RGB db. Because of our existing logic, those license searches would result in an error. We removed the logic around the error.

https://www.pivotaltracker.com/story/show/188341426

I noticed a flakey test when running this commit during CircleCI, I believe is was occurring b/c a disable industry was being selected. Logic was updated to prevent this. 

https://www.pivotaltracker.com/story/show/188408898
Remove unused randomOperatingPhaseId fn

## Code author checklist

- [x] I have rebased this branch from the latest main branch
- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation on the engineering documentation website
- [x] I have not used any relative imports
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] I have checked for and removed instances of unused config from CMS
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

**Review by somabadri — COMMENTED** [review]
Just one note: I see that you removed the `randomOperatingPhaseId` function because we don't use it anymore. Could you split it into its own commit so it's clear that it's not related to the home-based industry fix commit?"

File: api/src/client/dynamics/license-status/formatRegulatedBusinessDynamicsApplications.test.ts [hunk 1]
```diff
@@ -20,17 +19,4 @@ describe("formatRegulatedBusinessDynamicsApplications", () => {
...
-      professionNameAndLicenseType: "duplicate profession name and license type",
-    });
-    const checklistResult2 = generateLicenseStatusChecklistResult({
-      professionNameAndLicenseType: "duplicate profession name and license type",
-    });
-
-    expect(() => formatRegulatedBusinessDynamicsApplications([checklistResult1, checklistResult2])).toThrow(
-      DUPLICATE_LICENSE_TYPE_ERROR
-    );
-  });
```
Comment (alexander-littleton) [inline api/src/client/dynamics/license-status/formatRegulatedBusinessDynamicsApplications.test.ts:35]: [sand] - I wonder if it's worth expecting the inverse now as a happy path case to communicate to future devs that there can be duplicates.

**Review by alexander-littleton — APPROVED** [review]
LGTM

File: api/src/client/dynamics/license-status/formatRegulatedBusinessDynamicsApplications.test.ts (hunk 1 above)
Comment (FarazA22) [reply api/src/client/dynamics/license-status/formatRegulatedBusinessDynamicsApplications.test.ts]: the duplicates get over written, i added a test. thanks for the feedback

**Pull request merged**